src/identity/did-parser.ts

```diff
@@ -9,16 +9,16 @@ import {HcsDid} from "./hcs/did/hcs-did";
  * @return {@link HederaDid} instance.
  */
 export class DidParser {
-    public static parse(didString: string): HederaDid {
+    public static parse(didString: string): HcsDid {
```

Comment:
This appears to be a breaking change from the previous version (method response change), is this acceptable ?

Reply:
Same package, versioning imo takes care of notifying it's a major/breaking update. Devs have a choice to pull the latest version, or older.